aidants_connect_habilitation/templates/validation_form.html

```diff
@@ -1,59 +1,136 @@
-{% extends 'layouts/main-habilitation.html' %}
-{% load static form_extras %}
+{% extends 'layouts/main.html' %}
+{% load static ac_common form_extras dsfr_tags %}
 
 {% block title %}
-  Aidants Connect — Récapitulatif pour {{ organisation.name }}
+  Récapitulatif pour {{ organisation.name }} - Aidants Connect
+{% endblock %}
+
+{% block extracss %}
+  <link href="{% static 'css/generic-habilitation-request-profile-card.css' %}" rel="stylesheet">
 {% endblock %}
 
 {% block content %}
+
   <div class="fr-container">
-    {% include 'edito/_titre_formulaire_habilitation.html' %}
-    {% include 'layouts/_breadcrumbs.html' %}
-    <h2>Récapitulatif de la demande</h2>
-    <p class="subtitle">
-      Vous retrouverez ici les informations principales saisies dans le formulaire.
-      Il est possible de les modifier si besoin.
-    </p>
+    <h1>Récapitulatif de la demande</h1>
+    {% include '_recap_general_info.html' with show_edit_buttons=True %}
+    <div class="fr-alert fr-alert--info">
+      <h3 class="fr-alert__title">Une question ?</h3>
+      <p>
+        Si vous souhaitez ajouter une précision sur votre formulaire ou nous poser une question,
+        vous pouvez nous contacter à l’adresse suivante :<br />
+        {% mailto "contact@aidantsconnect.beta.gouv.fr" %}
+      </p>
+    </div>
+    <div class="fr-mt-6w">
+      <div class="flex flex-between">
+        <h2>Personnes impliquées</h2>
+        <div>
+          {% if organisation.status in organisation.Status.aidant_registrable %}
+            <a
+              id="add-aidants-btn"
+              class="fr-btn fr-btn--icon-left fr-icon-user-add-line fr-mb-2v"
+              href="{% url 'habilitation_organisation_add_aidants' issuer_id=organisation.issuer.issuer_id uuid=organisation.uuid %}"
+            >
+              Ajouter un aidant à la demande
+            </a>
+          {% endif %}
+        </div>
+      </div>
+      <p>Il vous est encore possible d’ajouter ou de supprimer des aidants de votre demande. Vous pouvez également
+        vérifier
+        les informations saisies ci-dessous.</p>
+      <div class="fr-grid-row fr-grid-row--gutters fr-mb-4v background-grey fr-p-3w">
```

Comment:
```suggestion
      <div class="fr-grid-row fr-grid-row--gutters fr-mb-4v fr-background-default--grey fr-p-3w">
```

Le DSFR frournit déjà des classes utils `fr-background-*`.